From: Jan Kara <jack@suse.cz>
References: fate#302681
Subject: ocfs2: Change file names of local quota files to be consistent
Patch-mainline: 2.6.29?

Change names of local quota files to be consistent with how other node-local
system files are named.

Signed-off-by: Jan Kara <jack@suse.cz>

Index: linux-2.6.27/fs/ocfs2/ocfs2_fs.h
===================================================================
--- linux-2.6.27.orig/fs/ocfs2/ocfs2_fs.h	2008-11-20 21:11:35.000000000 +0100
+++ linux-2.6.27/fs/ocfs2/ocfs2_fs.h	2008-11-20 21:13:16.000000000 +0100
@@ -366,8 +366,8 @@
 	[JOURNAL_SYSTEM_INODE]			= { "journal:%04d", OCFS2_JOURNAL_FL, S_IFREG | 0644 },
 	[LOCAL_ALLOC_SYSTEM_INODE]		= { "local_alloc:%04d", OCFS2_BITMAP_FL | OCFS2_LOCAL_ALLOC_FL, S_IFREG | 0644 },
 	[TRUNCATE_LOG_SYSTEM_INODE]		= { "truncate_log:%04d", OCFS2_DEALLOC_FL, S_IFREG | 0644 },
-	[LOCAL_USER_QUOTA_SYSTEM_INODE]		= { "aquota%04d.user", OCFS2_QUOTA_FL, S_IFREG | 0644 },
-	[LOCAL_GROUP_QUOTA_SYSTEM_INODE]	= { "aquota%04d.group", OCFS2_QUOTA_FL, S_IFREG | 0644 },
+	[LOCAL_USER_QUOTA_SYSTEM_INODE]		= { "aquota.user:%04d", OCFS2_QUOTA_FL, S_IFREG | 0644 },
+	[LOCAL_GROUP_QUOTA_SYSTEM_INODE]	= { "aquota.group:%04d", OCFS2_QUOTA_FL, S_IFREG | 0644 },
 };
 
 /* Parameter passed from mount.ocfs2 to module */
